### PR TITLE
Add a changelog

### PR DESCRIPTION
**Pull request by aboucaud**
Added skeleton, @kegl please fill in the blanks

### CODE REVIEW

**Comment by kegl** [comment]
Check the changelog (e.g. if nothing is removed, we still keep the header?), then merge. Then release 0.2.

**Comment by aboucaud** [comment]
I'll wait for @jorisvandenbossche before releasing 0.2 since I have never used `versioneer`

**Comment by kegl** [comment]
@jorisvandenbossche @aboucaud Maybe we should make version 0.1.1?

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
+# Changelog
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
+and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
+
+## [Unreleased]
+### Added
+- refactored README and wiki
+- `ramp_test_workflow` script: beautified score printing, optional pickling of trained models, optional saving `y_pred`, cv bagging
```
Comment (jorisvandenbossche) [inline CHANGELOG.md]: ramp_test_workflow -> ramp_test_submission ?

**Comment by jorisvandenbossche** [comment]
I think 0.2.0 is more logical instead of 0.1.1 since we added new functionality (0.1.x patch releases should only be for bug fixes), certainly if we state that we follow semantic versioning.

Using versioneer is not difficult, it is just making a git tag. But I will document the steps to make a release somewhere.

**Comment by kegl** [comment]
It's fine with me. I just want to make sure we can start communicating on our features, and that important ones don't get bundled with minor ones. The the upcoming features are:

1) Porting the ensembling from ramp-board to ramp-workflow. Besides it's useful, it will also be the last feature to make backend and starting kit testing identical, so hopefully testing will be less painful.
2) Launching `ramp_test_submission` on AWS, directly from prompt. It's almost done, works, though advanced features would be interesting to work out.

**Comment by jorisvandenbossche** [comment]
Page with release checklist is here: https://github.com/paris-saclay-cds/ramp-workflow/wiki/Releasing-ramp-workflow

**Pull request merged**